pkg/resource/aws/aws_route.go

```diff
@@ -30,6 +33,7 @@ func initAwsRouteMetaData(resourceSchemaRepository resource.SchemaRepositoryInte
 		val.DeleteIfDefault("local_gateway_id")
 		val.DeleteIfDefault("destination_cidr_block")
 		val.DeleteIfDefault("destination_ipv6_cidr_block")
+		val.DeleteIfDefault("destination_prefix_list_id")
```

Comment:
Not directly related to the bugfix sorry, but this will hide some useless noise.